README.md

```diff
@@ -105,43 +109,51 @@ storyblokInit({
 - Loads [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-react) for real-time visual updates.
 - Provides a `storyblokEditable` function to link editable components to the Storyblok Visual Editor.
 
-#### 1. Fetching Content
-
-Inject `getStoryblokApi`:
+For every component you've defined in your Storyblok space, call the `storyblokEditable` function with the blok content:
 
 ```js
-import { storyblokInit, apiPlugin, getStoryblokApi } from "@storyblok/react";
+import { storyblokEditable } from "@storyblok/react";
 
-storyblokInit({
-  accessToken: "YOUR_ACCESS_TOKEN",
-  // bridge: false,
-  // apiOptions: {  },
-  use: [apiPlugin],
-  components: {
-    page: Page,
-    teaser: Teaser,
-    grid: Grid,
-    feature: Feature,
-  },
-});
+const Feature = ({ blok }) => {
+  return (
+    <div {...storyblokEditable(blok)} key={blok._uid} data-test="feature">
+      <div>
+        <div>{blok.name}</div>
+        <p>{blok.description}</p>
+      </div>
+    </div>
+  );
+};
 
-const storyblokApi = getStoryblokApi();
-const { data } = await storyblokApi.get("cdn/stories", { version: "draft" });
+export default Feature;
 ```
 
-> Note: if you don't use `apiPlugin`, you can use your prefered method or function to fetch your data.
+Where `blok` is the actual blok data coming from [Storyblok's Content Delivery API](https://www.storyblok.com/docs/api/content-delivery?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-react).
+
+> Note: The `storyblokEditable` function works the same way for all the frameworks and components created.
+
 
-#### 2. Listen to Storyblok Visual Editor events
+### Getting Started
+
+**This SDK provides you the support to work with React, Next.js 12 (or Next.js 13 with no RSC), Next.js 13 (Full RSC) and Next.js (Partial RSC, with Live Editing). Depending upon these different frameworks and versions, the way to use the SDK and the functionalities it provides differ.**
+
+Below is the guide on how to use it with all the frameworks -
+
+
+## React
+
+The initalization remains the same when you work with React. You can intialze the SDK in the `index.js` file. Please refer to the 'Initialization' section above to read more.
 
-Use `useStoryblok` to get the new story every time is triggered a `change` event from the Visual Editor. You need to pass the `slug` as first param, and `apiOptions` as second param to update the new story. `bridgeOptions` (third param) is optional param if you want to set the options for bridge by yourself:
+### Fetching Content and Listening to Storyblok Visual Editor events
+
+For React, use `useStoryblok` to fetch the content as well as enable the live editing. You need to pass the `slug` as first parameter, and `apiOptions` as second parameter and `bridgeOptions` as third parameter which is optional if you want to set the options for bridge by yourself:
```

Comment:
```suggestion
Use `useStoryblok` to fetch the content as well as enable live editing. You need to pass the `slug` as the first parameter, `apiOptions` as the second parameter, and `bridgeOptions` as the third parameter, which is optional if you want to set the options for the bridge by yourself:
```

Reply:
also, should we explain what `apiOptions` and `bridgeOptions` mean and represent?

Reply:
Do you think this would be enough - Check the available [apiOptions](https://github.com/storyblok/storyblok-js-client#class-storyblok) (passed to `storyblok-js-client`) and [bridgeOptions](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-nuxt) (passed to the Storyblok Bridge). ?

Reply:
👌